apps/L1E/instagram/instagram-frontend/src/app/(main)/(auth)/signup/page.tsx

```
@@ -0,0 +1,148 @@
+
+    const form = useForm<z.infer<typeof formSchema>>({
+        resolver: zodResolver(formSchema),
+        defaultValues: {
+            email: "",
+            password: "",
+            fullname: "",
+            username: "",
+        },
+    });
+
+    const onSubmit = async (value: z.infer<typeof formSchema>) => {
```

ESLint

[error] 'value' is defined but never used. Allowed unused args must match /^_/u.

(no-unused-vars)